nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/PutCassandraQL.java

```diff
@@ -399,11 +434,13 @@ protected void setStatementObject(final BoundStatement statement, final int para
     @OnUnscheduled
     public void stop() {
         super.stop();
+        statementCache.clear();
     }
 
     @OnShutdown
     public void shutdown() {
         super.stop();
+        statementCache.clear();
```

Comment:
Given that we're calling this when the processor is stopped, I don't think there's a need for it here.